audit/projects/k8s-release/buckets/k8s-release-pull/logging.json

```diff
@@ -0,0 +1 @@
+{"logBucket": "k8s-infra-artifacts-gcslogs", "logObjectPrefix": "k8s-release-pull"}
```

Comment:
Expected, all the logging changes to `k8s-release-dev` et al are part of https://github.com/kubernetes/k8s.io/pull/2333